src/api/api.model.ts

```diff
@@ -1,44 +1,55 @@
 import { ApiProperty } from '@nestjs/swagger';
 
+// Patterns
 export const contractPattern =
   'ct_([23456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnpqrstuvwxyz]){49,50}';
 export const transactionPattern =
   'th_([23456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnpqrstuvwxyz]){49,50}';
 export const accountPattern =
   'ak_([23456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnpqrstuvwxyz]){49,50}';
 export const bigNumberPattern = '[1-9]+';
-export const usdValuePattern = '[1-9]+(.[1-9]{0,4})?';
-export const aeValuePattern = '[1-9]+(.[1-9]{0,18})?';
-export const percentPattern = '[1-9]+(.[1-9]+)?';
+export const usdValuePattern = '[1-9]+([.][1-9]{1,4})?'; // [.] instead of /. needed, because of the / being interpreted as a escape character in JavaScript resulting in incorrect pattern
```

Comment:
```suggestion
export const usdValuePattern = '[1-9]+([.][1-9]{1,4})?'; // [.] instead of \. needed, because of the \ being interpreted as a escape character in JavaScript resulting in incorrect pattern
```